traitsui/wx/dnd_editor.py

```diff
@@ -223,15 +222,11 @@ def _get_drag_data(self, data):
     def _unpickle(self, file_name):
         """ Returns the unpickled version of a specified file (if possible).
         """
-        fh = None
-        try:
-            fh = open(file_name, 'rb')
-            object = load(fh)
-        except:
-            object = None
-
-        if fh is not None:
-            fh.close()
+        with open(file_name, 'rb') as fh:
+            try:
+                object = load(fh)
+            except Exception:
```

Comment:
I want to suggest a more targeted `Exception` type here, but apparently `pickle.load` can produce a variety of exceptions (including `EOFError` and `UnpicklingError`), and I don't know what a sensible full list is.